### PR TITLE
Fix deprecated twig stuff usage

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Fixed Travis test with lowest tests.

ping @phiamo.

File: Twig/FormExtension.php [hunk 1]
```diff
@@ -29,8 +29,14 @@ class FormExtension extends \Twig_Extension
     public function getFunctions()
     {
         return array(
-            'form_help' => new \Twig_Function_Node('Symfony\Bridge\Twig\Node\SearchAndRenderBlockNode', array('is_safe' => array('html'))),
-            'form_tabs' => new \Twig_Function_Node('Symfony\Bridge\Twig\Node\SearchAndRenderBlockNode', array('is_safe' => array('html'))),
+            new \Twig_SimpleFunction('form_help', null, array(
+                'node_class' => 'Symfony\Bridge\Twig\Node\SearchAndRenderBlockNode',
+                'is_sage'    => array('html'),
```
Comment (isometriks) [inline Twig/FormExtension.php]: I assume this aren't all supposed to say `sage`?

Comment (Soullivaneuh) [inline Twig/FormExtension.php]: Well. No more wine anymore for me.

**Comment by Soullivaneuh** [comment]
@isometriks Review corrected.

**Comment by Richtermeister** [comment]
:+1: Yes please. Without this I'm running into Twig errors on 2.7 already.

**Comment by Soullivaneuh** [comment]
@isometriks Could we have a new tag for this? 41 commits since last one. Thanks. :+1:

**Comment by isometriks** [comment]
@Soullivaneuh all set. RC2 added. I'll have to speak with @phiamo about what else we need to finish for 3.0. Or if we should just get on with it and then release minor versions / patches from there.

**Pull request merged**